docs/Spark/gems/transform/dynamic-replace.md

```diff
@@ -0,0 +1,81 @@
+---
+title: DynamicReplace
+id: dynamic-replace
+description: Dynamically generate values depending on certain conditions
+tags:
+  - gems
+  - transform
+---
+
+<h3><span class="badge">Spark Gem</span></h3>
+
+Use the DynamicReplace gem to replace a set of values with dynamically generated values depending on certain conditions.
+
```

Comment:
Should the user add the Alteryx package dependency?
I wonder if this package is getting a new name.